Message bottom bar of following message is overlapping actions menu

## Steps to reproduce
1. add a emoji reaction to a message
2. add a emoji reaction to the message below
3. click the 3-dot menu of the first (notice it doesn't hide)
4. see the menu overlap

### Expected behaviour
* The 3-dot menu and reply button probably should hide
* the menu goes under the 3-dot menu and reply button below

### Actual behaviour
<img width="405" alt="Screenshot_20220408_170753" src="https://user-images.githubusercontent.com/551757/162471288-a3f6f7d6-9c1e-445f-aed5-18e1f959da84.png">

## Browser
* Mozilla Firefox latest
* Linux, latest

### Browser log
I had not tracked this and after a refresh this was gone. However, I can't add reactions anymore. Browser log below:

<details>

```

Content Security Policy: The page’s settings blocked the loading of a resource at inline (“script-src”). 2 moz-extension:40:65
No OC found index.js:46:4
Proxying an event bus of version 2.1.1 with 1.3.0 index.es.js:2337:14
JQMIGRATE: Migrate is installed, version 3.4.0 jquery-migrate.min.js:2:698
Proxying an event bus of version 2.1.1 with 1.3.0 index.es.js:2337:14
Proxying an event bus of version 2.1.1 with 1.3.0 index.es.js:2337:14
[WARN] viewer: Some mimes were ignored because they are not enabled in the server previews config 
Object { app: "viewer", uid: "jos", ignoredMimes: (4) […] }
ConsoleLogger.js:37:16
OCA.Viewer initialized Viewer.js:36:22
[INFO] viewer: No OCA.Files app found, viewer is now in standalone mode 
Object { app: "viewer", uid: "jos" }
ConsoleLogger.js:33:16
Proxying an event bus of version 2.1.1 with 1.3.0 index.es.js:2337:14
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
Proxying an event bus of version 2.1.1 with 1.3.0 index.es.js:2337:14
jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. globals.js:62:15
Proxying an event bus of version 2.1.1 with 1.3.0 major.js:2:14
Setting current user App.vue:328
You need to fill either the text or the ariaLabel props in the button component. 4 Button.js:1:21577
Detected browser Firefox 98 (98.0) AuthorAvatar.vue:1:524
jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 2 globals.js:62:15
Proxying an event bus of version 2.1.1 with 1.3.0 index.es.js:2337:14
Loading theming data for notification bell styling App.vue:150
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. globals.js:62:15
Registering notifications container as a menu App.vue:160
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 2 globals.js:62:15
Notifications permissions granted App.vue:395
Polling interval updated to 30000 App.vue:355
Started background fetcher as session_keepalive is enabled App.vue:217
Proxying an event bus of version 2.1.1 with 1.3.0 index.es.js:2337:14
Proxying an event bus of version 2.1.1 with 1.3.0 index.es.js:2337:14
[ERROR] viewer: The following mime is already registered 
Object { app: "viewer", uid: "jos", mime: "text/csv", handler: {…} }
ConsoleLogger.js:41:16
[INFO] viewer: 6 viewer handlers registered 
Object { app: "viewer", uid: "jos", handlers: (6) […] }
ConsoleLogger.js:33:16
session heartbeat polling started session-heartbeat.js:103:9
[DEBUG] unified-search: Unified Search initialized with the following providers 
Object { 0: {…}, 1: {…}, 2: {…}, 3: {…}, 4: {…}, 5: {…}, 6: {…}, 7: {…}, 8: {…}, 9: {…}, … }
ConsoleLogger.js:29:8
X-Nextcloud-Talk-Hash initialised:  a292726faf3110bdc3a116b7e22796ed4245c5e9 talkHashStore.js:78:3
You need to fill either the text or the ariaLabel props in the button component. Button.js:1:21577
Got notification data App.vue:318
Polling interval updated to 30000 App.vue:355
You need to fill either the text or the ariaLabel props in the button component. 3 Button.js:1:20945
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. globals.js:62:15
You need to fill either the text or the ariaLabel props in the button component. Button.js:1:20945
New notification, animating the bell icon App.vue:192
Joined signaling.js:217:2
You need to fill either the text or the ariaLabel props in the button component. Button.js:1:21577
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 2 globals.js:62:15
Use of drawWindow method from CanvasRenderingContext2D is deprecated. Use tabs.captureTab extensions API instead https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/tabs/captureTab pagedata-script.js:1:12465
[WASM] Loading model of size: 249792 talk-JitsiStreamBackgroundEffect.worker.worker.js:2:33640
INFO: Created TensorFlow Lite XNNPACK delegate for CPU. talk-JitsiStreamBackgroundEffect.worker.worker.js:2:33640
clearLastReadMessage because of isSticky token=pqdix2cn MessagesList.vue:706
You need to fill either the text or the ariaLabel props in the button component. Button.js:1:21577
The request has been canceled 
Object { message: "canceled" }
MessagesList.vue:534
Joined signaling.js:217:2
You need to fill either the text or the ariaLabel props in the button component. 2 Button.js:1:21577
The request has been canceled 
Object { message: "canceled" }
MessagesList.vue:534
Joined signaling.js:217:2
You need to fill either the text or the ariaLabel props in the button component. 6 Button.js:1:21577
Error: Request failed with status code 400
    exports createError.js:16
    exports settle.js:17
    C xhr.js:66
messagesStore.js:1004:3
You need to fill either the text or the ariaLabel props in the button component. 4 Button.js:1:21577
Error: Request failed with status code 400
    exports createError.js:16
    exports settle.js:17
    C xhr.js:66
messagesStore.js:1004:3
jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 804 globals.js:62:15
XHRGEThttps://tech-preview.nextcloud.com/ocs/v2.php/apps/spreed/api/v4/room/mws3kp6q/participants?includeStatus=true
[HTTP/1.1 200 OK 154ms]

setVisualLastReadMessageId token=mws3kp6q id=514 MessagesList.vue:586
jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
You need to fill either the text or the ariaLabel props in the button component. Button.js:1:21577
jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 8 globals.js:62:15
You need to fill either the text or the ariaLabel props in the button component. Button.js:1:21577
jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 8 globals.js:62:15
You need to fill either the text or the ariaLabel props in the button component. Button.js:1:21577
jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 8 globals.js:62:15
You need to fill either the text or the ariaLabel props in the button component. Button.js:1:21577
jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 8 globals.js:62:15
You need to fill either the text or the ariaLabel props in the button component. Button.js:1:21577
jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 8 globals.js:62:15
You need to fill either the text or the ariaLabel props in the button component. Button.js:1:21577
jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
You need to fill either the text or the ariaLabel props in the button component. 3 Button.js:1:21577
jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 352 globals.js:62:15
XHRPOSThttps://tech-preview.nextcloud.com/ocs/v2.php/apps/spreed/api/v1/reaction/mws3kp6q/458
[HTTP/1.1 400 Bad request 90ms]

Error: Request failed with status code 400
    exports createError.js:16
    exports settle.js:17
    C xhr.js:66
    exports xhr.js:78
    exports xhr.js:15
    exports dispatchRequest.js:58
    request Axios.js:112
    e Axios.js:147
    exports bind.js:9
    e messagesService.js:136
    c dist.js:1
    _invoke dist.js:1
    S dist.js:1
    ae guestNameStore.js:93
    o guestNameStore.js:93
    ie guestNameStore.js:93
    ie guestNameStore.js:93
    de messagesService.js:135
    a messagesStore.js:981
    c dist.js:1
    _invoke dist.js:1
    S dist.js:1
    ge messagesService.js:149
    o messagesService.js:149
    Ae messagesService.js:149
    Ae messagesService.js:149
    addReactionToMessage messagesStore.js:981
    v vuex.esm.js:736
    dispatch vuex.esm.js:490
    dispatch vuex.esm.js:393
    handleReactionClick MessageButtonsBar.vue:414
    click MessageButtonsBar.vue:1
    VueJS 37
    isReactionsMenuOpen Message.vue:1
    VueJS 4
    openReactionsMenu MessageButtonsBar.vue:438
    VueJS 16
messagesStore.js:1004:3
jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 368 globals.js:62:15
setVisualLastReadMessageId token=mws3kp6q id=514 2 MessagesList.vue:586
jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
XHRGEThttps://tech-preview.nextcloud.com/ocs/v2.php/apps/spreed/api/v4/room?includeStatus=true
[HTTP/1.1 200 OK 191ms]

XHRGEThttps://tech-preview.nextcloud.com/ocs/v2.php/apps/notifications/api/v2/notifications
[HTTP/1.1 304 Not Modified 145ms]

No new notification data received App.vue:323
Polling interval updated to 30000 App.vue:355
jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 184 globals.js:62:15
XHRGEThttps://tech-preview.nextcloud.com/ocs/v2.php/apps/spreed/api/v1/chat/mws3kp6q?setReadMarker=0&lookIntoFuture=1&lastKnownMessageId=514&includeLastKnown=0
XHRGEThttps://tech-preview.nextcloud.com/ocs/v2.php/apps/spreed/api/v1/chat/mws3kp6q?setReadMarker=0&lookIntoFuture=1&lastKnownMessageId=514&includeLastKnown=0
XHRGEThttps://tech-preview.nextcloud.com/ocs/v2.php/apps/spreed/api/v3/signaling/mws3kp6q
XHRGEThttps://tech-preview.nextcloud.com/ocs/v2.php/apps/spreed/api/v3/signaling/mws3kp6q
```

</details>

now I get other errors, can't join some chats anymore...

Maybe click around in an account on tech-preview, some login details here:
https://cloud.nextcloud.com/index.php/f/3406050

--- a/src/components/MessagesList/MessagesGroup/Message/MessageButtonsBar/MessageButtonsBar.vue
+++ b/src/components/MessagesList/MessagesGroup/Message/MessageButtonsBar/MessageButtonsBar.vue
@@ -477,10 +477,9 @@ export default {
 	right: 14px;
 	bottom: -4px;
 	position: absolute;
-	z-index: 100000;
 	background-color: var(--color-main-background);
 	border-radius: calc($clickable-area / 2);
-	box-shadow: 0 0 4px 0px var(--color-box-shadow);
+	box-shadow: 0 0 4px 0 var(--color-box-shadow);
 
 	& h6 {
 		margin-left: auto;

